File: Python/Euler5.py
#2520 is the smallest number that can be divided by each
#of the numbers from 1 to 10 without any remainder.
#
#What is the smallest number that is evenly divisible by
#all of the numbers from 1 to 20?

current = 2520

while(not all(current % x == 0 for x in range(1,20))):
	current += 20
	
print current

File: Node.js/Euler5.js
/*
2520 is the smallest number that can be divided by each of the numbers from 1 to 10 without any remainder.

What is the smallest positive number that is evenly divisible by all of the numbers from 1 to 20?
*/

function IsDivisibleByAll(val)
{
	for(var z = 1; z <= 20; z++)
	{
		if(val % z != 0) return false;
	}
	
	return true;
}

var curr = 20;

while (!IsDivisibleByAll(curr)) curr += 20;

console.log(curr);